ci/gpu/build.sh

```diff
@@ -59,13 +59,12 @@ gpuci_conda_retry install -y \
                "cuspatial=$MINOR_VERSION.*" \
                "dask-cudf=$MINOR_VERSION.*" "dask-cuda=$MINOR_VERSION.*" \
                "numba>=0.51.2" \
-               "bokeh<=2.2.3" \
                "rapids-build-env=$MINOR_VERSION.*" \
                "rapids-notebook-env=$MINOR_VERSION.*"
 
 # https://docs.rapids.ai/maintainers/depmgmt/ 
-# conda remove -f rapids-build-env rapids-notebook-env
-# conda install "your-pkg=1.0.0"
+conda remove -f rapids-build-env rapids-notebook-env
```

Comment:
```suggestion
conda remove --force rapids-build-env rapids-notebook-env
```

whoops. looks like this flag needs to be updated to use `--force` instead of just `-f`.

Reply:
thanks, trying it